portfolio/src/main/java/com/google/sps/servlets/AuthServlet.java

```diff
@@ -37,12 +37,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     boolean loginStatus = userService.isUserLoggedIn();
 
     String authUrl;
+    String username = null;
 
     if (!loginStatus) {
-      authUrl = userService.createLoginURL("/"); //TODO: Redirect to the page where the reuqest was made.
+      authUrl = userService.createLoginURL("/"); //  TODO: Redirect to the page where the reuqest was made.
```

Comment:
[Nit] Put 2 spaces after the semicolon for include comments e.g.
`x = 5;  // 5 is a really cool number`

Reply:
You can have just 1 space after the "//" BTW.

Reply:
I fixed this in PR #19, but it still shows here because the PR is a draft.